### ISSUE TITLE
Wrong bucket name used for S3 events

### ISSUE THREAD

**Issue by rkistner** [serverless/serverless#1977]
# This is a Bug Report

## Description

When configuring an S3 event to trigger a function, an integer is added to the bucket name.

Example config:

```
functions:
  myfunction:
    handler: my.handler
    events:
      - s3:
          bucket: some_bucket
          event: s3:ObjectCreated:*
```

For this example, "some_bucket0" would be used as the bucket name, instead of just "some_bucket". It seems to append the index number of the event configuration.

Seems to be caused by this code: https://github.com/serverless/serverless/blob/81e933b194ab09ecd202820e5ea41f38afd5a66d/lib/plugins/aws/deploy/compile/events/s3/index.js#L37 (lines 37 and 42).

## Additional Data

* ***Serverless Framework Version you're using:*** 1.0.0-beta.2
* ***Operating System:*** Ubuntu 16.04, node v4.2.3

**Comment by flomotlik** [comment]
This issue will be resolved with #1951 so I'm closing here. Thanks for reporting!

**Comment by rkistner** [comment]
Thanks!
